definitions/npm/xorshift_v1.x.x/flow_v0.32.0-/xorshift_v1.x.x.js

```diff
@@ -0,0 +1,18 @@
+declare module 'xorshift' {
+  declare class $npm$xorshift {
```

Comment:
If you use `declare module.exports`, then this class will not be automatically exported.. you can theoretically leave out the `$npm$` part